fjord/__init__.py

```diff
@@ -1,7 +1,16 @@
+# First we import absolute imports from the future, so that our celery.py
+# module will not clash with the library
+from __future__ import absolute_import
+
 import os
 import site
 import sys
 
+from fjord.manage_utils import monkeypatch
+
+
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'fjord.settings')
```

Comment:
The code path of using the celery command as `$ celery -A fjord worker -l info` which is the suggested way now.